apps/frontend/src/app/pages/company/applications-job/applications-job.component.html

```diff
@@ -0,0 +1,72 @@
+<div class="container mx-auto min-h-screen px-3">
+  <div class="pb-4 pt-6 flex justify-between flex-wrap gap-4">
+    <div
```

Comment:
![image](https://github.com/Venture-Miner/wagademy/assets/111710515/f396f6c6-f6d5-41db-b605-cff5061ba4c9)

![image](https://github.com/Venture-Miner/wagademy/assets/111710515/26dc204b-9c52-4d96-98b0-b1420779c5df)


These buttons are way taller than the ones on figma, also, the selected color on figma is different